.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I'm curious, do we have a standard ruleset that we're using across Electron-related projects?

Reply:
Doesn't seem like it. For example, [`@electron/get`](https://github.com/electron/get/blob/master/.prettierrc.json).